### PR TITLE
Book: Add instructions on how to download precompiled release

### PR DESCRIPTION
**Pull request by bauerj**
This adds instructions on how to download releases from Github. This is especially useful on the Raspberry Pi as compiling takes a lot of time there.

Related: #2583

### CODE REVIEW

File: docs/the_nimbus_book/src/keep-updated.md [hunk 1]
```diff
@@ -2,7 +2,23 @@
...
+
+To update to the latest version, either:
+
+## Download the binary
+
+Open the [Nimbus eth2 releases page](https://github.com/status-im/nimbus-eth2/releases/latest) and copy the link for the file that works on your system.
+
+```
+wget <insert download link here>
+tar xvf nimbus-eth2_Linux_arm64v8*.tar.gz -C nimbus-eth2
```
Comment (stefantalpalaru) [inline docs/the_nimbus_book/src/keep-updated.md]: Make this `tar -xzf`.

File: docs/the_nimbus_book/src/pi-guide.md [hunk 2]
```diff
@@ -258,25 +249,22 @@ Run the following command to install `screen`:
...
-### 14. Build the beacon node
+Run this in your home directory to download nimbus-eth2:
 
-Change into the directory and build the beacon node.
 ```
-cd nimbus-eth2
-make nimbus_beacon_node
+mkdir nimbus-eth2
+wget <insert download link here>
+tar xvf nimbus-eth2_Linux_arm64v8*.tar.gz -C nimbus-eth2
```
Comment (stefantalpalaru) [inline docs/the_nimbus_book/src/pi-guide.md]: `tar -xzf`

File: docs/the_nimbus_book/src/pi-guide.md [hunk 3]
```diff
@@ -240,15 +240,6 @@ Follow [this guide](https://www.tomshardware.com/how-to/boot-raspberry-pi-4-usb)
...
-### 11. Install Nimbus dependencies
-
-You'll need to install some packages (`git`) in order for Nimbus to run correctly.
-
-To do so, run:
-```
-sudo apt-get install git
-
-```
 ### 12. Install Screen
```
Comment (stefantalpalaru) [inline docs/the_nimbus_book/src/pi-guide.md]: Shouldn't this be section 11 now?

**Comment by bauerj** [comment]
Thanks, I fixed that now.

**Comment by stefantalpalaru** [comment]
There are two more instances of "tar xvf" that need changing.
When you rebase it on top of the "unstable" branch, you need to edit this PR and change the branch it targets in here.

**Comment by bauerj** [comment]
Okay, I rebased it on unstable and changed the base branch.

**Pull request merged**